lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -1033,6 +1033,14 @@ def _create_deployment_parameters(
         ):
             arm_parameters.use_availability_sets = True
 
+        if any(
+            x.capability.has_feature(features.Infiniband.name())
```

Comment:
@squirrelsc I had to change this away from what you had suggested. Please let me know if this is stylistically bad.